src/RadixDlt.NetworkGateway.GatewayApi/Services/IEntityStateQuerier.cs

```diff
@@ -76,7 +76,7 @@ public sealed record PageRequest(GlobalAddress Address, int Offset, int Limit);
 
     public sealed record ResourceVaultsPageRequest(GlobalAddress Address, GlobalAddress ResourceAddress, int Offset, int Limit);
 
-    Task<GatewayModel.StateEntityDetailsResponseItem> EntityDetailsItem(GlobalAddress address, bool aggregatePerVault, GatewayModel.LedgerState ledgerState, CancellationToken token = default);
+    Task<GatewayModel.StateEntityDetailsResponseItem> EntityDetailsItem(GlobalAddress address, bool aggregatePerVault, GatewayModel.StateEntityDetailsOptInProperties? optInProperties, GatewayModel.LedgerState ledgerState, CancellationToken token = default);
```

Comment:
Make it mandatory property and use something like `request.OptIns ?? XxxOptIns.Default` in controller?